source/manage/cloud-data-residency.rst

```diff
@@ -0,0 +1,18 @@
+Mattermost Cloud Data Residency
+===============================
+
+Mattermost Cloud resides in the ``aws-us-east-1`` region, located in Virginia, United States. The following customer data will be stored at rest in this data center when using Mattermost Cloud:
+
+- Channels, messages, replies and channel membership information
+- Boards and cards
+- Playbooks and event runs
+- User accounts and profiles 
+- Files (ex., images, docs, etc.) uploaded to the Mattermost Service
+- Plugin, app- or bot-generated messages and files
+- Data used to measure user count, usage, and revenue
+- Data used for analytics and to measure quality of service, ex. sanitized logs
+- IDs generated by Mattermost on behalf of the customer
+
+Additional data regions are considered, including ``aws-eu-central-1`` located in Frankfurt, Germany.
+
+If you require your data to reside in an area outside of the United States, please contact the product team via feedback-cloud[at]mattermost.com, or consider `deploying one of our Self-Hosted options <https://mattermost.com/deploy>`_ that provides full control of your data. You may also work with `one of our European partners <https://mattermost.com/partners>`_ for deploying and hosting your Mattermost server.
```

Comment:
Question: Is there a specific reason why the feedback email account isn't selectable and is written with [at] instead?

Reply:
I left it as-is because I assumed it was to counter bots/spam but I am 0/5 and can make it selectable. :)

Reply:
Sounds good!